src/main/scala/org/phenoscape/kb/Main.scala

```diff
@@ -151,9 +151,10 @@ object Main extends HttpApp with App {
                 }
               } ~
               path("label") {
-                parameters('iri.as[IRI]) { (iri) =>
+                parameters('iri.as[Seq[IRI]]) { (iri) =>
```

Comment:
I think you should only accept a single IRI here, especially since only one label can be returned anyway. Our conversation about this may have been confusing.